### PR TITLE
Medical Orderly (experimental role)

### PR DESCRIPTION
**Pull request by Apogee-dev**
[role]
orderly

Experimental, may need reversion later if it turns out to have been a bad idea. Coded in response to the fact that this idea has been consistently popping up again and again since at least 2019; We may as well give it a try.

Adds the Orderly as a new medical job. Two parts janitor, one part clerk, and one part bouncer, Orderlies focus on keeping medbay clean and relatively shitter-free while the doctors tunnel vision on whichever corpse has been placed in front of them this time. Clean up an inevitable ocean of blood, morgue forgotten bodies, maintain medical records, and flush the occasional assistant down disposals. It's still a medical job, though. They aren't sec, and will still need to step up to the plate and fix some dead spessmen eventually. Traitor orderlies, like the medical doctor, get a discount on the syndicate wheelchair.

Adds Suxameth, a paralytic agent made from toxin and chlorine. Suxameth slows down the victim for five seconds before stunning him (not knocking him out). At 10 units, the stun lasts roughly 10 seconds. It overdoses at 21 units, causing moderate oxy damage, and is purged by dylovene.

Additionally, adds a medical waste garbage bag to the biohazard closet of every medbay. Unlike the regular trash bag, the medical waste bag can only carry trash, shards, and medical items, including organs. It is also sterile.

Orderlies are limited to 2 job slots like the paramedic. They spawn behind Medbay's front desk with the orderly jumpsuit, pockets containing a bar of soap and a 10u Suxameth autoinjector, a briefcase containing six more injectors and a lollipop dispenser, and a straightjacket in their hands.

:cl:
 * rscadd: Added Orderly job for medbay, responsible for cleaning, record maintenance, and other busywork the doctors usually don't have time for.

### CODE REVIEW

**Comment by DrSnips** [comment]
You could put the straight jacket in their bag to start. It's just a one way street action when they remove it

**Comment by Apogee-dev** [comment]
> 
> 
> You could put the straight jacket in their bag to start and when they take it out it's a one way street action.

There's a specific reason I placed it in their hands to start. Spawn in and press E and you'll see what I mean.

**Comment by DrSnips** [comment]
> > You could put the straight jacket in their bag to start and when they take it out it's a one way street action.
> 
> There's a specific reason I placed it in their hands to start. Spawn in and press E and you'll see what I mean.

🧠  Fair enough.

**Comment by Optimism333** [comment]
> Clean up an inevitable ocean of blood, morgue forgotten bodies, maintain medical records, and flush the occasional assistant down disposals. 

What a shame all the Nurses are too busy giving johnson inspections to do any of the first three things on the list.

On a more serious note, while I have some reservations about this job being in a weird place in terms of general usefulness, I'm willing to see how it works out. Maybe it will be a good addition to the medbay roster.

**Comment by Apogee-dev** [comment]
> On a more serious note, while I have some reservations about this job being in a weird place in terms of general usefulness, I'm willing to see how it works out. Maybe it will be a good addition to the medbay roster.

Yeah, at the end of the day this is experimental. There's a significant chance it just won't work out after all- perhaps it needs to be its own job with more limited slots, perhaps the entire idea's a dud. But if it does work out, it could be a nice add to medbay.

**Comment by Kurfursten** [comment]
What I would rather see is some actual equipment designed for this role to separate it from MD. Anyone can grab a bar of soap and syringe of stoxin pretty easily. As some critics have brought up, why don't we see people with the Nurse title doing these things? We have cultural preconceptions about what a nurse is - and it's not similar to what nurses do in reality. I'm not sure most people have a preconception of what an orderly does at all. I think in a world where this role was fully engaged with, it would probably be a separate role so that it could have access to its own vendor or locker (an office isn't really necessary), HOP can mark as high priority, and intro text/wiki page to explain exactly what your deal is supposed to be.

Here are the kinds of things I would associate with an orderly if they had a whole locker:
* Special gloves that give +shove chance and shove resistance. You know the problem with immediately harm-stabbing someone with sleep toxin? (1) It doesn't work immediately. (2) It escalates things pretty fast because STOXIN sucks to get stabbed with. You have to really want them gone now. -- Sleep toxin is thematic for orderlies but not a great option for peacefully removing someone who is mostly just trespassing. You don't have to get rid of it but I would reserve this for clowns eating the fridge. **The first priority of an orderly should be to remove people from medbay, instead of trying to detain them with a straight jacket or chloral - fundamentally different from officers who attempt arrests**.
* Orderly belt. Like the detective belt, this has more specialty in terms of what it holds. Holds medical stuff like syringes and pills for sedation, health analyzer, but also holds your janitorial gear like soap, spray bottle, trash bag, etc.
* Possibly some kind of special combo HUD that gives both medhud and sechud but without most of the features of sechud shades. No flash protection, no shift-click to change prisoner records, no job icon, no loyalty implant detection. Just show if someone is wanted so that you know they don't belong in medbay.
* A special firstaid kit like paramedics have. Like the paramedic kit for internal bleeding, this is actually a very thematic way to show someone the injectors that define their role
* Trash bag (maybe a special medical waste one that can grab large objects like lost limbs), soap
* Lollipop dispenser, like the borg has

What I don't want to see for orderlies: (since we're at the position of trying to define what this role even is)
* Ranged stuns like tasers, guns, or RSG - keeping orderly limited discourages validhunting and therefore defending medbay instead of playing the role for access to guns. It also encourages calling for real security if shit gets real, like a wizard. Orderly is aimed at stemming the tide of visitor inflow, not ninjas.
* Anything that makes them seem like security. No flashproof shades, body armor, sec radio, access to security records, access to the brig. Don't add a prison cell to medbay.
* Insulated gloves, box of lights, multitool, etc. - these sorts of lower priority repairs should remain in the domain of calling the janitor or engineering staff, like how a high threat enemy involves calling for sec instead.

I think there is enough here to make a real role out of it. I think people have the interest in bringing in a new role, particularly if it doesn't involve remapping medbay to add a room.

**Comment by Eneocho** [comment]
Orderlies could get some injectors with 1u Amasec (about 5 seconds of paralysis) or any other paralytic chem so they can disable the guy (but not mute them) for a few moments to throw them out instead of using Sleeptox, which has an inherent validhunting stigma.

File: code/datums/outfit/medical.dm [hunk 1]
```diff
@@ -103,11 +104,22 @@
 			slot_head_str = list(
 				"Surgeron" = /obj/item/clothing/head/surgery/blue,
```
Comment (Eneocho) [inline code/datums/outfit/medical.dm]:
```suggestion
				"Surgeon" = /obj/item/clothing/head/surgery/blue,
```
You should fix this typo while you're at it.

**Comment by Kurfursten** [comment]
> 
> 
> Orderlies could get some injectors with 1u Amasec (about 5 seconds of paralysis) or any other paralytic chem so they can disable the guy (but not mute them) for a few moments to throw them out instead of using Sleeptox, which has an inherent validhunting stigma.

Had to check my contacts because this is a good Eneocho post

**Comment by Eneocho** [comment]
> 
> Had to check my contacts because this is a good Eneocho post

Make sure you don't fall asleep with them on though.

**Comment by BomberBro** [comment]
Orderly completes the conceptual devolution of Medbay into Assistant v. Security started by Brig Medic

**Comment by Optimism333** [comment]
I agree with what Kurfursten said, and I also think that if the job is going to be experimental, it might be worth locking it to a single available slot on round-start. To better gauge how a few individual players use the role round to round, and if it is helpful.

If it turns out to be a useful addition, maybe another slot can be opened in a later pr. I think two orderly slots should be more than enough for a singular medbay.

**Comment by Apogee-dev** [comment]
I'm extremely leery of giving the orderly better stuns for a few reasons. I really don't want to overemphasize the orderly's shitter-removal role; he's a janitor first, a clerk second, and a bouncer dead last. This is also the reason orderly is an MD alt title and not a separate role. The second is that _because_ he's an MD alt title, the amount of stun items needs to be kept a close eye on since he spawns in with them. This is the reason I elected not to give the orderly a flash like I'd originally considered.

>shoveglubbs

I don't think this one's a good idea simply because there's no real countermeasure for shoves outside of being fat or having a shield and gloves can't be disarmed.

>Amasec injectors

Actually not a completely terrible idea, though I'm still a bit shy of introducing too many new items here. Giving him a first aid kit with these would be a nice parallel with paramedic.... but I feel like, conceptually, this (and the other orderly-specific items kurfurst suggested) would only work if Orderly was indeed a separate role instead of an alt title. This also ties into....

>limit the role to 1 slot

This is impossible to do while Orderly is an alt title, since you can't set a different limit for alt titles than the job they're based on. Part of the experiment is, really, to see if this idea _can_ work as an alt title or if it really justifies having a separate role.
Most of the suggestions made above seem tooled more towards the idea of orderly as a job rather than orderly as a title, and I honestly am starting to think it may be better to take the separate-job route just to make it _possible_ to have that specialized gear.

File: code/datums/outfit/medical.dm (hunk 1 above)
Comment (Apogee-dev) [reply code/datums/outfit/medical.dm]: I fixed this in the first commit and was (rightfully) yelled at about atomicity, especially since this is an experimental PR that stands a good chance of being reverted. I'll fix it later in its own PR.

**Comment by Kurfursten** [comment]
I think (somewhat counterintuitively) it's easier to have it as a separate slot even as an experiment. This allows for
* Stricter control of access
* Limited slots
* A separate wiki link
* Separate intro text

Moreover, if it gets removed, old Orderly legacy titles will stick around on MD slots. But a removed job stays removed.

**Comment by Apogee-dev** [comment]
> Moreover, if it gets removed, old Orderly legacy titles will stick around on MD slots. But a removed job stays removed.

I was already in the process of moving it over to a distinct job slot but this in particular is a good point.

**Comment by gurfan** [comment]
I was going to write a github essay but @BomberBro said it better than I ever could. The amalgamation of different roles is going to reduce interaction and is bad for the game. This is taking the 'chloral and disposal' part of medbay and making it into a role. These guys are just going to be the people who silently chloral/stox/shove/disarm you, but now they have even MORE tools to do so.

**Comment by Apogee-dev** [comment]
> 
> 
> I was going to write a github essay but @BomberBro said it better than I ever could. The amalgamation of different roles is going to reduce interaction and is bad for the game. This is taking the 'chloral and disposal' part of medbay and making it into a role. These guys are just going to be the people who silently chloral/stox/shove/disarm/whatever you for standing in medbay.

It's been getting intermittently requested for at least the past two years, and the players requesting it have by and large been medbay players. There's a reason this is experimental- if it does prove to have a detrimental effect after a month or so I'll open a revert request myself. But it's worth trying out, or we'll never stop hearing about it.

**Comment by Apogee-dev** [comment]
Major update. Orderly is now a separate job from MD, with 1 slot. Orderlies spawn with a briefcase containing 6 1u Amasec autoinjectors and a lollipop dispenser, and another injector has replaced the sleeptox syringe in their pocket. For reference, 1 unit of Amasec equates to roughly 5 seconds of stun- it's not a knockout, so the "patient" can still speak. Think of them as being kind of like a one-use, non-refillable stun baton. Hopefully the limited supply and relatively short duration will discourage orderlies from abusing them. I've also added a biohazard trash bag that has a more limited list of accepted items than the regular trash bag, but can pick up limbs and organs. They're also completely sterile, so they won't spread diseases. These aren't directly available yet, but one will be mapped into the biohazard closet in medbay storage.

**Comment by Apogee-dev** [comment]
On a related note, stun chemicals in general need some work. Having to use a bartender chemical in order to get something even close to a reasonable stun without going into a full-fledged chloral knockout is kind of lame. Something that slows the patient down for a couple seconds before making him collapse, still conscious and able to speak, would be ideal, but that's out of scope for this particular PR.

**Comment by gurfan** [comment]
Why are all these fancy tools necessary? Why is this role necessary? Security exists for kicking people out of medbay. The janitor exists for cleaning up medbay. If neither are around, doctors already have more than enough tools available to fill in for a janitor/secoff. There are at least 4 space cleaner bottles in every single medbay, and two chemistry dispensers that let you make even more cleaner. Doctors  already have an infinite supply of chloral and sleeptoxin, plus cryotubes for dunking, plus DNA scanners for jailing, plus syringe guns for shooting shitters, and so on. Any normal doctor can do what an orderly can do. When the orderly is done kicking people out of medbay with their fancy injectors (why play as an orderly otherwise) what will they do afterwards? Cleaning up medbay and kicking out shitters takes maybe a minute. After that, they're just a shitty doctor with less access. Do you expect them to just afk at the desk until they get the next opportunity to stab someone with an injector?

**Comment by Apogee-dev** [comment]
> Why is this role necessary? Security exists for kicking people out of medbay. The janitor exists for cleaning up medbay. If neither are around, doctors already have more than enough tools available to fill in for a janitor/secoff. 

Mostly because secoffs and MDs are usually too busy to actually do those tasks, and the janitor lacks the access. Security's typically putting out other fires and can't be bothered to come deal with greyshirts breaking into chemistry for the fourteenth time unless the round's slower than usual; when they do loiter around medbay it's because the station's already well into SHTF territory (and they usually just get in the way). Similarly doctors are typically too busy attending to patients (on an average round anyway) to actually clean up medbay and pay any heed to tailgaters. There's a reason medbay's cleanliness is usually a good measure of the overall state of a round. As for the janitor, he can't get in by himself and the doctors rarely bother to call him in unless it's exceptionally slow (again, because they're usually busy with patients). 

Obviously, deadpop is a different beast entirely, and ideally i'd like to block this role from even becoming available until the station hits a minimum population/projected population threshold (honestly, the same could be said for a handful of other roles), but we don't have a system for that.

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 2]
```diff
@@ -124,3 +124,16 @@
 		icon_state = "biofoam1"
 	else
 		icon_state = "biofoam0"
+
+/obj/item/weapon/reagent_containers/hypospray/autoinjector/sedative_injector
+	name = "sedative injector"
```
Comment (Eneocho) [inline code/modules/reagents/reagent_containers/hypospray.dm]: Amasec is a paralytic not a sedative though.

**Comment by Eneocho** [comment]
You should change the PR's title to reflect it being a new experimental role instead of an alt-title. @Apogee-dev

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 3]
```diff
@@ -124,3 +124,16 @@
...
+	desc = "A small, single-use device used to administer small amounts of sedative."
+	amount_per_transfer_from_this = 15
+	volume = 15
+	flags = FPRINT
+
+/obj/item/weapon/reagent_containers/hypospray/autoinjector/sedative_injector/New()
+	..()
+	reagents.remove_reagent(DOCTORSDELIGHT, 30)
+	reagents.add_reagent(AMASEC, 1)
+	return
```
Comment (Kurfursten) [inline code/modules/reagents/reagent_containers/hypospray.dm]: No need for a trailing return here

It's also hacky that we need to remove the doctorsdelight because of a lack of an abstract type but I won't insist on you changing that

**Comment by TGFH** [comment]
I agree with all of what Gurfan (and Harris) said. Pointless role for a problem that can already be solved ingame. Call security or arm the medbay staff if the tide is that bad. You've got chloral syringes etc. and the CMO already has two flashes. Medbay doesn't need a dedicated janny, since thye can just open the door and let the janny in like everyone else.
Most of all I dislike this role because it seems like it's trying to make medbay more cloistered and self sufficient in a game that is meant to be social. I dislike brig medic for similar reasons.

**Comment by SonixApache** [comment]
`armband/med` | `armband/medgreen` ?

File: code/datums/outfit/medical.dm [hunk 4]
```diff
@@ -103,11 +103,16 @@
...
+			slot_s_store_str = list(
+				"Emergency Physician" = /obj/item/device/flashlight/pen,
+				"Surgeon" = /obj/item/device/flashlight/pen,
+				"Medical Doctor" = /obj/item/device/flashlight/pen,
+			),
 		),
 		/datum/species/plasmaman = list(
 			slot_ears_str = /obj/item/device/radio/headset/headset_med,
 			slot_w_uniform_str = list(
+				"Orderly" = /obj/item/clothing/under/rank/orderly,
```
Comment (Kurfursten) [inline code/datums/outfit/medical.dm]: This can be removed

File: code/datums/outfit/medical.dm [hunk 5]
```diff
@@ -103,11 +103,16 @@
 			slot_head_str = list(
 				"Surgeron" = /obj/item/clothing/head/surgery/blue,
 			),
-			slot_s_store_str = /obj/item/device/flashlight/pen,
+			slot_s_store_str = list(
```
Comment (Kurfursten) [inline code/datums/outfit/medical.dm]: go back to the old way since it's all the same

File: code/datums/outfit/medical.dm [hunk 6]
```diff
@@ -370,6 +375,74 @@
...
+			slot_head_str = /obj/item/clothing/head/helmet/space/vox/civ/medical,
+			slot_wear_mask_str =  /obj/item/clothing/mask/breath/vox,
+			slot_l_store_str = /obj/item/weapon/reagent_containers/syringe/stoxin,
+			slot_r_store_str = /obj/item/weapon/soap,
+		),
+	)
+
+	race_items_to_collect = list(
+		/datum/species/vox/ = list(
+			"Emergency Physician" = /obj/item/clothing/suit/storage/fr_jacket,
```
Comment (Kurfursten) [inline code/datums/outfit/medical.dm]: these titles don't exist for this job

File: code/datums/outfit/medical.dm [hunk 7]
```diff
@@ -370,6 +375,74 @@
...
+			"Surgeon" =  /obj/item/clothing/suit/storage/labcoat,
+			"Medical Doctor" =  /obj/item/clothing/suit/storage/labcoat,
+		),
+		/datum/species/plasmaman/ = list(
+			"Emergency Physician" = /obj/item/clothing/suit/storage/fr_jacket,
```
Comment (Kurfursten) [inline code/datums/outfit/medical.dm]: same

File: code/game/objects/items/weapons/storage/bags.dm [hunk 8]
```diff
@@ -55,6 +55,22 @@
 		icon_state = "trashbag3"
 		slowdown = 1.8
 
+/obj/item/weapon/storage/bag/trash/bio
+	name = "hazardous waste bag"
+	desc = "A heavy-duty sterilized garbage bag for handling infectious medical waste and sharps."
+	icon_state = "biobag0"
+	item_state = "biobag"
+
+	sterility = "10"
```
Comment (Kurfursten) [inline code/game/objects/items/weapons/storage/bags.dm]: must be NUM not STRING

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 9]
```diff
@@ -124,3 +124,15 @@
 		icon_state = "biofoam1"
 	else
 		icon_state = "biofoam0"
+
+/obj/item/weapon/reagent_containers/hypospray/autoinjector/paralytic_injector
```
Comment (Kurfursten) [inline code/modules/reagents/reagent_containers/hypospray.dm:128]: if you can, it would be nice if this had an alternate sprite and inhand because it *is* a weapon

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 10]
```diff
@@ -124,3 +124,15 @@
...
+	name = "paralytic injector"
+	desc = "A small, single-use device used to administer small amounts of paralytic agent."
+	amount_per_transfer_from_this = 15
+	volume = 15
+	flags = FPRINT
+
+/obj/item/weapon/reagent_containers/hypospray/autoinjector/paralytic_injector/New()
+	..()
+	reagents.remove_reagent(DOCTORSDELIGHT, 30)
+	reagents.add_reagent(AMASEC, 1)
```
Comment (Kurfursten) [inline code/modules/reagents/reagent_containers/hypospray.dm]: I like the effect Amasec has, but it's a bardrink. It would be nice if there was an actual chem that behaved similarly to Amasec that they could mix up in chemistry. Since you have a little time to work on this PR (as American Lights has mapping priority), I suggest you make a new chem, which has a few advantages.
* Mixable in chemistry.
* Better immersion instead of being an Imperium bar drink.
* Amasec can stack up to really high paralysis if they get injected with a lot because of AdjustStunned. Instead, if we use SetStunned, they will stop being stunned quickly after they are purged of the chemical.
* By adding an overdose we can discourage orderlies from putting a lot in people, in the same way chloral has to be measured if you aren't trying to kill someone.

```
/datum/reagent/suxameth
	name = "Suxameth"
	id = SUX
	description = "A name for Suxamethonium chloride. A medical full-body paralytic preferred because it is easy to purge."
	reagent_state = REAGENT_STATE_LIQUID
	color = "#664300" //rgb: 102, 67, 0
	overdose_am = 11

/datum/reagent/suxameth/on_mob_life(var/mob/living/M)
	if(..())
		return 1

	M.SetStunned(2)

/datum/reagent/suxameth(var/mob/living/M)
	M.adjustOxyLoss(4) //Very rapid deterioration if they go over 10 units
```

with a recipe
(Toxin is easily obtained from a nanomed vendor but not easily found in most other places, which means this will be harder to ghettochem -- you can still do it if you're clever though)

```
/datum/chemical_reaction/suxameth
	name = "Suxameth"
	id = SUX
	result = SUX
	required_reagents = list(TOXIN = 1, CHLORINE = 1)
	result_amount = 2
```

I also recommend changing the injectors to contain 5 units, so it lasts 10 ticks with 1 default stab, 2 stabs is safe, but 3 stabs will overdose.

Comment (Apogee-dev) [reply code/modules/reagents/reagent_containers/hypospray.dm]: I considered adding a new reagent earlier considering the overall lack of a proper paralytic agent that wouldn't be overwhelmingly broken, but I was concerned about going over the top. I like the look of the suxameth suggestion here. I'll go ahead and add it.

**Comment by Apogee-dev** [comment]
Sux will need some tweaking, because I really don't want this to just be Chloral 2.0. Ideally I'd like a single pen of it to stun someone for roughly as long as a stunbaton. They're basically non-refillable hyposprays, so that means there aren't very many way to prevent getting jabbed with one. With that in mind, I'm also going to reduce the number of injectors available to the orderly now that chemistry can make more paralytic. I've got concerns about the overdose too- for something that's meant to be a safer, less obnoxious alternative to chloral, these kill much faster in their current state and have a lower overdose threshold. Obviously a paralytic agent _should_ paralyze the diaphragm (and cause oxyloss) when overdosed, but I need to spend some time dialing in the threshold and the damage to a point that feels right.

Also I'll go back and fix the mapping bits after eurolights gets merged.

**Comment by Kurfursten** [comment]
See fit to tweak it as need arises. Time based overdose is fine. I do not see it replacing Chloral as a killing method because
* Chloral requires a similar amount but is easier to make (toxin is hard to come by outside medbay)
* Chloral fully knocks out the victim whereas this allows them to call for help
* Chloral inflicts toxin damage which is nastier, whereas oxygen damage you can walk off easily as long as you have a good amount of blood

The hope with the OD is to prevent orderlies from just pumping someone with assloads of it like stoxin.
In order to get it to last the same length as a stun baton I think what you want is to keep everything as-is but add `custom_metabolism = 0.6`. Try this in testing and see how it compares.

**Review by Kurfursten — APPROVED** [review]

**Comment by Apogee-dev** [comment]
Yeah, I've been tweaking the metabolism, injector amounts, etc. I actually think a 21u overdose threshold (with appropriate adjustments to the other stats) may be more appropriate, because at 11u you could theoretically get somebody killed with a single syringe. I'm also adding in antitoxin purging sux the same way it purges other toxins and I'm working on getting sux to slow you down for a few seconds before actually stunning. Hopefully.

**Comment by Eneocho** [comment]
>
>toxin is hard to come by outside medbay

Is it though? Burnt food has toxin, food touched by a roach has toxins, a braindead with organ damage produces toxin, electrolizing PBG or Insecticide, etc.

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 11]
```diff
@@ -4181,6 +4181,23 @@
...
+	overdose_am = 11
+
+/datum/reagent/suxameth/on_mob_life(var/mob/living/M)
+	if(..())
+		return 1
+
+	M.SetStunned(2)
+
+/datum/reagent/suxameth/on_overdose(var/mob/living/M)
+	M.adjustOxyLoss(4) //Very rapid deterioration if they go over 10 units
```
Comment (Eneocho) [inline code/modules/reagents/Chemistry-Reagents.dm]: I feel that on overdose it should do the same amount of o2 damage as when you're in a room without oxygen, since it stops you from breathing as per your explanation, it doesn't do oxygen damage by itself.

**Comment by Apogee-dev** [comment]
> 
> 
> > toxin is hard to come by outside medbay
> 
> Is it though? Burnt food has toxin, food touched by a roach has toxins, a braindead with organ damage produces toxin, electrolizing PBG or Insecticide, etc.

Toxin's only marginally harder to get than the base ingredients for, say, chloral. It's a step in difficulty but only a small one. That's part of why I'm concerned about making sure sux isn't a better stun than chloral is.

**Comment by Eneocho** [comment]
> 
> That's part of why I'm concerned about making sure sux isn't a better stun than chloral is.

It won't be chloral 2 simply because it doesn't silence the target. If nothing else, an option could be to make it easier/faster to make than chloral to incentivize its use. All stuns are disliked, but one that doesn't KO you is far preferable (on the victim's side). Nothing is more boring that getting shot with a 15u gloral syringe and having to wait 2 minutes without being able to even insult your attacker.

**Comment by Apogee-dev** [comment]
> 
> 
> > That's part of why I'm concerned about making sure sux isn't a better stun than chloral is.
> 
> It won't be chloral 2 simply because it doesn't silence the target. If nothing else, an option could be to make it easier/faster to make than chloral to incentivize its use. All stuns are disliked, but one that doesn't KO you is far preferable.

I think the recipe's fine- unlike chloral, you get a good quantity out of it, and toxin's easy to get within medbay itself. I just need to keep tuning the effects. You're dead right about silence being the worst part of chloral, and that's the entire reason i didn't just give the orderly gloral to begin with.

**Comment by Eneocho** [comment]
>I just need to keep tuning the effects.

I still think it should have some implicit incentive for it to be made instead of chloral. The recipe is far better than chloral, I'll give you that (60u at 30u energy cost vs 20u at 100u energy cost), but ease of manufacture (or lack thereof) has never stopped the chloral users. Most people who'd rely on chloral would still use it simply because it silences the target as well. Sux needs a bigger edge over chloral for it to become the preferred stunchem.

**Comment by Kurfursten** [comment]
To answer your question: yes, you can get toxin from food and burnt food, but at a rate of 1 per food it's going to take a while to get 15 units and quite frankly you're going to have to break your microwave and fix it several times. You can electrolyze some cargo plantkiller or insecticide, true, but then you still need to centrifuge it out and electrolyze salt etc. This is a lot of energy, effort, and spent batteries and at this point you probably deserve a stun chem for your work. Or you ask the mechanic for a chem dispenser and make chloral in 5 clicks without the fuss.

Anyway arguing about this is a little redundant since I think we all agree here that chloral will remain the stun chem king.

**Comment by Apogee-dev** [comment]
> Sux needs a bigger edge over chloral for it to become the preferred stunchem.

I'm not trying to make sux the preferred stunchem, is the thing. That's always going to be chloral because of the silencing effect- trying to balance sux to match or exceed chloral's utility is a terrible idea in general, considering how often people complain about chloral. Personally I'm of the belief that chloral could use a nerf- or, god forbid, a recipe removal- but this isn't the PR to discuss that in depth.

tl;dr chloral is too strong and we don't need another stunchem that's just as good

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 12]
```diff
@@ -4181,6 +4183,31 @@
...
+	color = "#CFC5E9" //rgb: 207, 197, 223
+	data = null
+	flags = CHEMFLAG_DISHONORABLE
+	overdose_am = 21
+	custom_metabolism = 1
+
+/datum/reagent/suxameth/on_mob_life(var/mob/living/M)
+	if(..())
+		return 1
+		if(isnull(data))
```
Comment (DamianX) [inline code/modules/reagents/Chemistry-Reagents.dm]: this is not indented correctly

**Comment by Kurfursten** [comment]
Is this ready?

**Comment by Apogee-dev** [comment]
> 
> 
> Is this ready?

I just had to add the spawns and biobag to waystation, but it should be ready to go now.
